src/Medidata.ZipkinTracer.Core/ZipkinConfig.cs

```diff
@@ -32,12 +29,25 @@ public string SpanProcessorBatchSize
 
         public string DontSampleListCsv
         {
-            get {  return ConfigurationManager.AppSettings["mAuthWhitelist"];}
+            get {  return ConfigurationManager.AppSettings["uriBlacklist"];} // TODO: refactor this later if it is being used
```

Comment:
@kenyamat Modified key name and documentation here --> 18b2ddd
Please check if change is ok.
Thanks